### DISCUSSION TITLE
About the java modular system

### DISCUSSION THREAD

**Discussion by TheBlindM** [quarkusio/quarkus#25433] [Q&A]
Why no one uses the java modular system

**Comment by maxandersen** [comment]
Because it is not obvious you get an advantage ? 

Why do you think someone should use javas module system ?

**Reply by TheBlindM** [reply]
Because this is a feature of java, I want to try it on quarkus. The modular system should be the trend. In theory, it can help you make a more streamlined jar, but we are all used to the current way of writing.
